astrokat/test/astrokat_testutils.py

```diff
@@ -0,0 +1,42 @@
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2019 SKA SA. All rights reserved.                               #
+#                                                                             #
+# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
+# WRITTEN PERMISSION OF SKA SA.                                               #
+###############################################################################
+import logging
+import os
+from six.moves import StringIO
+
+from astrokat import observe_main, simulate
+
+
+__all__ = ['yaml_path', 'LoggedTelescope']
+
+
+def yaml_path(file_path):
+    tests_path = os.path.abspath(os.path.dirname(__file__))
+    """Convenience method for finding the yaml's file absolute path."""
+    yaml_file = os.path.abspath(os.path.join(tests_path, file_path))
+    assert os.path.isfile(yaml_file)
+    return yaml_file
+
+
+class LoggedTelescope(observe_main.Telescope):
+
+    string_stream = StringIO()
```

Comment:
Looking at the usage, it isn't clear that this contains the user_logger output.  Maybe `user_logger_stream`?
Would also be nice to add docstring or comment explaining the purpose of this class.